### PR TITLE
Add roles for grid

### PR DESCRIPTION
**Pull request by Reettaphant**
- Add roles needed for cropper and imgops in grid @kenoir

### CODE REVIEW

File: roles/imgops-logrotate/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+- name: Move imgops config files into logrotate folder
+  copy: src=logrotate dest=/etc/logrotate.d/imgops
```
Comment (cb372) [inline roles/imgops-logrotate/tasks/main.yml]: You can set the owner and group as part of the `copy`, meaning you can delete the `Change owner to root` task.

```
copy: src=logrotate dest=/etc/logrotate.d/imgops owner=root group=root
```

**Comment by cb372** [comment]
Looks good apart from one small simplification

**Comment by Reettaphant** [comment]
I've made that change now, thanks!

**Comment by cb372** [comment]
👍  I'll deploy Amigo with these changes now

**Pull request merged**